rpg/load_game_map.py

```diff
@@ -9,11 +9,12 @@
 import arcade
 
 from rpg.sprites.character_sprite import CharacterSprite
-from rpg.constants import TILE_SCALING
+from rpg.constants import TILE_SCALING, STARTING_MAP
 from rpg.sprites.path_following_sprite import PathFollowingSprite
 from rpg.sprites.random_walking_sprite import RandomWalkingSprite
 
 GOD_MODE = False
+LOAD_ALL_MAPS = False
```

Comment:
One thing to consider is to move these settings to environment variables, which we shouldn't do now, but good to keep in mind

Pydantic makes this really easy with something like:

```py
from pydantic import BaseSettings

class _Settings(BaseSettings):
    GOD_MODE: bool = False
    LOAD_ALL_MAPS: bool = False

SETTINGS = _Settings()
```

Then you can set:

`LOAD_ALL_MAPS=true poery run python -m rpg`

(But again, this would be for future work)

Reply:
These are both variables that I was planning to remove later on